.github/CHANGELOG.md

```diff
@@ -457,13 +490,13 @@ and requirements-ci.txt (unpinned). This latter would be used by the CI.
   Temporary backward compatibility has been added to support the use of `_stepsize` for one
   release cycle. `update_stepsize` method is deprecated.
   [(#1625)](https://github.com/PennyLaneAI/pennylane/pull/1625)
-  
+
 
 <h3>Bug fixes</h3>
 
 * Fix bug with norm and Jax tracers (jit) when using `QubiStateVector`.
   [(#1649)](https://github.com/PennyLaneAI/pennylane/pull/1649)
```

Comment:
This is not descriptive enough - suggest rewriting it to inform a reader how this impacts their workflow:
```suggestion
* Fixed a bug with where `@jax.jit` would fail on a QNode that used `qml.QubiStateVector`.
  [(#1649)](https://github.com/PennyLaneAI/pennylane/pull/1649)
```